python/requirements-test.txt

```diff
@@ -5,3 +5,4 @@ pickle5; python_version == "3.6" or python_version == "3.7"
 pytest
 pytest-lazy-fixture
 pytz
+pycryptodomex
```

Comment:
Might be nice to use `cryptography`, since I'm told it's a more secure library than `pycryptodomex`. Not necessary because these are just tests, but helpful in case users copy/paste test code to start their own KMS client.

Reply:
+1 for `cryptography`. I was a bit surprised to see `pycryptodomex` in the first place.